apiv6/localhost_apiserver/main.go

```diff
@@ -7,26 +7,38 @@ import (
 	"time"
 
 	"github.com/gorilla/mux"
-
 	cloudexportstub "github.com/kentik/community_sdk_golang/apiv6/localhost_apiserver/cloudexport/go"
 	syntheticsstub "github.com/kentik/community_sdk_golang/apiv6/localhost_apiserver/synthetics/go"
 )
 
 func main() {
 	address, cloudexportFilePath, syntheticsFilePath := readAddressStoragePaths()
 
-	router := makeRouter(cloudexportstub.NewCloudExportRepo(cloudexportFilePath), syntheticsstub.NewSyntheticsRepo(syntheticsFilePath))
+	router := makeRouter(
+		cloudexportstub.NewCloudExportRepo(cloudexportFilePath),
+		syntheticsstub.NewSyntheticsRepo(syntheticsFilePath),
+	)
 
 	log.Printf("Server started, address %s", address)
 	log.Fatal(http.ListenAndServe(address, router))
 }
 
 func readAddressStoragePaths() (address string, cloudexport string, synthetics string) {
 	flag.StringVar(&address, "addr", ":8080", "Address for the server to listen on")
-	flag.StringVar(&cloudexport, "cloudexport", "CloudExportStorage.json", "JSON file path for the server to read and write the cloud export data to")
-	flag.StringVar(&synthetics, "synthetics", "SyntheticsStorage.json", "JSON file path for the server to read and write the synthetics data to")
+	flag.StringVar(
+		&cloudexport,
+		"cloudexport",
+		"CloudExportStorage.json",
+		"JSON file path for the server to read and write the cloud export data to",
+	)
+	flag.StringVar(
+		&synthetics,
+		"synthetics",
+		"SyntheticsStorage.json",
+		"JSON file path for the server to read and write the synthetics data to",
+	)
 	flag.Parse()
-	return
+	return address, cloudexport, synthetics
```

Comment:
It is interesting that the code worked before without any value being returned  ;-).

Reply:
It was a naked return: https://tour.golang.org/basics/7 - possible, but unreadable